src/Mod/Sketcher/App/PythonConverter.cpp

```diff
@@ -195,6 +195,30 @@ std::string PythonConverter::convert(const std::string& doc,
     return constraintlist;
 }
 
+template<typename T>
+std::string makeSplineInfoArrayString(const std::vector<T>& rInfoVec)
+{
+    std::stringstream stream;
+    if constexpr (std::is_same_v<T, Base::Vector3d>) {
+        for (const auto& rInfo : rInfoVec) {
+            stream << "App.Vector(" << rInfo.x << ", " << rInfo.y << "), ";
+        }
+    }
+    else {
+        for (const auto& rInfo : rInfoVec) {
+            stream << rInfo << ", ";
+        }
+    }
+
+    std::string res = stream.str();
+    // remove last comma and add brackets
+    int index = res.rfind(',');
+    res.resize(index);
+    res.insert(0, 1, '[');
+    res.append(1, ']');
+    return res;

```

Comment:
This probably could be rewritten as:

```suggestion
return fmt::format("[{}]", stream.str());
```

As @AjinkyaDahale noticed - there is no need to remove the comma at the end. This will be cleaner and easier to maintain.

Reply:
Great advice!  I'm fully agree on replace inster & append code.  But about remove last comma, I'm still think we need to remove it.  As @AjinkyaDahale mentioned the final string will printed in pthon console.  Although keeping it the converting still works but since it finally will be a pthon script removing it will making more readable.
<img width="1913" alt="Screenshot 2025-05-07 123122" src="https://github.com/user-attachments/assets/11ed76d7-5330-49ed-bd99-67942590dc3e" />